packages/env/lib/create-docker-compose-config.js

```diff
@@ -0,0 +1,55 @@
+module.exports = function createDockerComposeConfig(
+	pluginPath,
+	pluginName,
+	pluginTestsPath
+) {
+	const commonVolumes = `
+      - ${ pluginPath }/:/var/www/html/wp-content/plugins/${ pluginName }/
+      - ${ pluginPath }${ pluginTestsPath }/e2e-tests/mu-plugins/:/var/www/html/wp-content/mu-plugins/
```

Comment:
This isn't very useful outside of Gutenberg.

Reply:
https://github.com/WordPress/gutenberg/pull/17668#discussion_r330671160